docker-compose.yml

```
@@ -0,0 +1,19 @@
+version: '3.2'
+
+services:
+    build-scan-db:
+        image: mongo
```

```suggestion
        image: mongo:4.4.5-bionic
```
Specifies mongo version to latest stable